### PR TITLE
Docs: review of the runner/nrunner section

### PR DESCRIPTION
**Pull request by clebergnu**
Part of the contributor's guide.

Fixes: https://github.com/avocado-framework/avocado/issues/4194
Signed-off-by: Cleber Rosa <crosa@redhat.com>

### CODE REVIEW

File: docs/source/guides/contributor/chapters/runners.rst [hunk 1]
```diff
@@ -26,23 +26,25 @@ There are a number of reasons for introducing a different architecture
...
 
 * Test execution limited to the same machine, given that the
-  communication between runner and test is a Python queue (the remote
-  runner plugins actually execute an Avocado Job remotely, with all
-  the overhead and complications that it brings)
-* Test execution is limited to a single test at a time (non-parallel)
+  communication between runner and test is a Python queue (the
+  "remote" runner plugins Avocado once had would actually execute an
+  Avocado Job remotely, with all the overhead and complications that
+  it brings)
```
Comment (willianrampazzo) [inline docs/source/guides/contributor/chapters/runners.rst:38]: The sentence in parenthesis is confusing.

Comment (clebergnu) [reply docs/source/guides/contributor/chapters/runners.rst:38]: Hummm... suggestions?

Comment (beraldoleal) [reply docs/source/guides/contributor/chapters/runners.rst:38]: Do we really need this sentence? What about removing it? And avoid references to old code/plugins? I mean: I know that this as part of an RFC document but now we could consolidate this to make it easy for new contributors. 

What do you think?

Comment (willianrampazzo) [reply docs/source/guides/contributor/chapters/runners.rst:38]: I second @beraldoleal; we don't need to refer to old, removed, code/plugins. Remove the sentence in parenthesis will avoid confusion.

If you prefer to refer it, my suggestion is to go straight to the point, like: "Once, Avocado had remote runner plugins, with all the overhead and complication brought by them."

Comment (clebergnu) [reply docs/source/guides/contributor/chapters/runners.rst]: > I second @beraldoleal; we don't need to refer to old, removed, code/plugins. Remove the sentence in parenthesis will avoid confusion.
> 
> If you prefer to refer it, my suggestion is to go straight to the point, like: "Once, Avocado had remote runner plugins, with all the overhead and complication brought by them."

Pushed 4fcfc53856b4e96c3d95fee5d6e186bc7f25d380 with that sentence removed.

**Review by beraldoleal — APPROVED** [review]

**Review by willianrampazzo — APPROVED** [review]

**Pull request merged**